packages/rocketchat-livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -59,6 +59,43 @@ Template.livechatCurrentChats.events({
 		instance.filter.set(filter);
 		instance.limit.set(20);
 	},
+	'click .delete-all'() {
+		LivechatRoom.find({
```

Comment:
Since we have filters in the interface and its implementation is based on client-side data, I think the best approach  would be delete records according the current filters in the `LivechatRoom` collection.